expat/tests/fuzz/parse_fuzzer.cc

```diff
@@ -0,0 +1,77 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+// Modifications
+//
+//                          __  __            _
+//                      ___\ \/ /_ __   __ _| |_
+//                      / _ \\  /| '_ \ / _` | __|
+//                     |  __//  \| |_) | (_| | |_
+//                      \___/_/\_\ .__/ \__,_|\__|
+//                               |_| XML parser
+//
+// Copyright (c) 2017 Expat development team
+// Licensed under the MIT license:
+//
+// Permission is  hereby granted,  free of charge,  to any  person obtaining
+// a  copy  of  this  software   and  associated  documentation  files  (the
+// "Software"),  to  deal in  the  Software  without restriction,  including
+// without  limitation the  rights  to use,  copy,  modify, merge,  publish,
+// distribute, sublicense, and/or sell copies of the Software, and to permit
+// persons  to whom  the Software  is  furnished to  do so,  subject to  the
+// following conditions:
+//
+// The above copyright  notice and this permission notice  shall be included
+// in all copies or substantial portions of the Software.
+//
+// THE  SOFTWARE  IS  PROVIDED  "AS  IS",  WITHOUT  WARRANTY  OF  ANY  KIND,
+// EXPRESS  OR IMPLIED,  INCLUDING  BUT  NOT LIMITED  TO  THE WARRANTIES  OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN
+// NO EVENT SHALL THE AUTHORS OR  COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR  OTHER LIABILITY, WHETHER  IN AN  ACTION OF CONTRACT,  TORT OR
+// OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
+// USE OR OTHER DEALINGS IN THE SOFTWARE.
```

Comment:
I'm not sure if mixing BSD and MIT in a single file is a good idea: Which lines arre gong to be BSD, which ones MIT?
I'd prefer to just have BSD in this file. What do you think?

Reply:
changed to back to the original BSD (as this file is taken from oss-fuzz repo). in general there is no problem mixing bsd and mit license afaik.